### PR TITLE
add command line option for EXPIRE_TILES_MAX_BBOX instead of hard coded value of 20000

### PR DESCRIPTION
**Pull request by JesseCrocker**
Default value is still 20000

new lines in --help --verbose output"
```
       -B|--expire-bbox-size Max size for a polygon to expire the whole polygon,
                             not just the boundary.
```

### CODE REVIEW

**Comment by lonvia** [comment]
Looks good to me. @pnorman any objection at adding yet another command line parameter?

**Comment by pnorman** [comment]
I'd like to avoid using up a short letter for something that's going to be infrequently used

**Comment by JesseCrocker** [comment]
I have removed the single letter option.

File: options.cpp [hunk 1]
```diff
@@ -168,6 +169,8 @@ namespace
     Expiry options:\n\
        -e|--expire-tiles [min_zoom-]max_zoom    Create a tile expiry list.\n\
        -o|--expire-output filename  Output filename for expired tiles list.\n\
+       -B|--expire-bbox-size Max size for a polygon to expire the whole polygon,\n\
+                             not just the boundary.\n\
```
Comment (lonvia) [inline options.cpp]: You forgot to adapt the documentation. While you change that, can you please also adapt the number of options in README.md and docs/usage.md.

**Pull request merged**